### ISSUE TITLE
Error in part 9 - Class inheritance - Quiz "inheritance"

### ISSUE THREAD

**Issue by TraKar** [rage/java-programming#141]
I thought that the answer was 2, but submitted it and it was wrong. Ran the code in my IDE and got 2 as the answer also, however the quiz says the correct answer is 8.

**Comment by EricTGoto** [comment]
Yep same issue. There are too many small mistakes in part 2 of the course

**Comment by ZdravekSprite** [comment]
There is error in program code that is in quiz:
`number = counter.lisaaLukuun(number);`
should be:
`number = counter.addToNumber(number);`
so may be is intended that one of the two
`number = superCounter.subtractFromNumber(number);`
is
number = superCounter.addToNumber(number);
then correct answer will be 8

**Comment by ZdravekSprite** [comment]
Waiting for the "Coming Soon" parts to open, I switched to the Finnish version of the course and there it is:

public static void main(String[] args) {
    Laskin laskin = new Laskin();
    Laskin superLaskin = new SuperLaskin();
    int luku = 3;
    luku = superLaskin.vahennaLuvusta(luku);
    luku = superLaskin.lisaaLukuun(luku);
    luku = laskin.lisaaLukuun(luku);
    System.out.println(luku);
}

which means that an error occurred while translating the task it should be:

public static void main(String[] args) {
    Counter counter = new Counter();
    Counter superCounter = new SuperCounter();
    int number = 3;
    number = superCounter.subtractFromNumber(number);
    number = superCounter.addToNumber(number);
    number = counter.addToNumber(number);
    System.out.println(number);
}

**Comment by thanosades** [comment]
Sad to see this issue open for almost 2 years and not yet fixed.

**Comment by thanosades** [comment]
> > Sad to see this issue open for almost 2 years and not yet fixed.
> 
> It was reported in 24th of April 2020 so better prepare the 3 year old birthday party.

And the funny thing is, I ran into it again this year revisiting Java, only to see my comment in this thread. Oopsies!
